A(z) gexiv2-0.10.9.original/.ipc_in f치jl egy fifo, viszont a(z) gexiv2-0.10.9/.ipc_in f치jl egy fifo
A(z) gexiv2-0.10.9.original/.ipc_out f치jl egy fifo, viszont a(z) gexiv2-0.10.9/.ipc_out f치jl egy fifo
diff -ur gexiv2-0.10.9.original/docs/reference/Makefile.am gexiv2-0.10.9/docs/reference/Makefile.am
--- gexiv2-0.10.9.original/docs/reference/Makefile.am   2018-11-18 10:37:00.000000000 +0100
+++ gexiv2-0.10.9/docs/reference/Makefile.am    2019-01-02 11:22:45.094668216 +0100
@@ -71,7 +71,7 @@
 GTKDOC_LIBS=$(top_builddir)/lib@PACKAGE_NAME@.la
 
 # This includes the standard gtk-doc make rules, copied by gtkdocize.
-include $(top_srcdir)/gtk-doc.make
+#include $(top_srcdir)/gtk-doc.make
 
 # Comment this out if you want 'make check' to test you doc status
 # and run some sanity checks
diff -ur gexiv2-0.10.9.original/Makefile.am gexiv2-0.10.9/Makefile.am
--- gexiv2-0.10.9.original/Makefile.am  2018-11-18 10:37:00.000000000 +0100
+++ gexiv2-0.10.9/Makefile.am   2019-01-02 11:22:28.566858633 +0100
@@ -14,7 +14,6 @@
 vapidir          = $(datadir)/vala/vapi
 
 DISTCHECK_CONFIGURE_FLAGS = \
-   --enable-gtk-doc \
    --with-python2-girdir='$${libdir}/python2/dist-packages/gi/overrides' \
    --with-python3-girdir='$${libdir}/python3/dist-packages/gi/overrides' \
    --enable-introspection

